tests/test_transformers/test_capper.py

```diff
@@ -0,0 +1,15 @@
+from sklego.preprocessing import ColumnsCapper
+from sklearn.base import TransformerMixin
+import pandas as pd
+
+
+test_data = pd.DataFrame({'var1':range(101)})
+capper = ColumnsCapper(quantile_thresholds=0.9)
```

Comment:
you're bound to write more than one test for this transformer. so it might be better to add this initialisation in the test itself.